subsys/mgmt/hawkbit/shell.c

```diff
@@ -18,49 +18,7 @@ static void cmd_run(const struct shell *sh, size_t argc, char **argv)
 	ARG_UNUSED(argv);
 
 	shell_fprintf(sh, SHELL_INFO, "Starting hawkBit run...\n");
-	switch (hawkbit_probe()) {
-	case HAWKBIT_UNCONFIRMED_IMAGE:
-		shell_fprintf(
-			sh, SHELL_ERROR,
-			"Image is unconfirmed."
-			"Rebooting to revert back to previous confirmed image\n");
-		hawkbit_reboot();
-		break;
-
-	case HAWKBIT_CANCEL_UPDATE:
-		shell_fprintf(sh, SHELL_INFO,
-			      "hawkBit update Cancelled from server\n");
-		break;
-
-	case HAWKBIT_NO_UPDATE:
-		shell_fprintf(sh, SHELL_INFO, "No update found\n");
-		break;
-
-	case HAWKBIT_OK:
-		shell_fprintf(sh, SHELL_INFO, "Image Already updated\n");
-		break;
-
-	case HAWKBIT_UPDATE_INSTALLED:
-		shell_fprintf(sh, SHELL_INFO, "Update Installed\n");
-		hawkbit_reboot();
-		break;
-
-	case HAWKBIT_DOWNLOAD_ERROR:
-		shell_fprintf(sh, SHELL_INFO, "Download Error\n");
-		break;
-
-	case HAWKBIT_NETWORKING_ERROR:
-		shell_fprintf(sh, SHELL_INFO, "Networking Error\n");
-		break;
-
-	case HAWKBIT_METADATA_ERROR:
-		shell_fprintf(sh, SHELL_INFO, "Metadata Error\n");
-		break;
-
-	default:
-		shell_fprintf(sh, SHELL_ERROR, "Invalid response\n");
-		break;
-	}
+	hawkbit_autohandler_once();
```

Comment:
This is not equal to what we had before - `shell_fprintf` would 'reply' to the shell instance where the command came from, `LOG_*` would be sending to whatever log backends configured with the target verbosity.

So if only LOG UART backend is enabled and command is coming from shell_telnet, the user will not be able to see the messages.